apps/site/pages/docs/api-extensions.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: '  Overview'
+---
+
+import { Steps, Step } from 'nextra-theme-docs'
+
+<header className="hero">
+
+# API Extensions
+
+In this guide you will learn how to extends the FastStore API schema.
+</header>
+
+Although the [FastStore API](https://v1.faststore.dev/reference/api/faststore-api) provides a complete [GraphQL schema for ecommerce](https://v1.faststore.dev/reference/api/queries), some stores may need to access other, more specific, information.
+
+You should not add requests to other APIs to your storefront since this may [impact site performance](https://v1.faststore.dev/how-to-guides/faststore-api/fetching-api-data#best-practices-for-fetching-data). For those cases, it is possible to extend the FastStore API schema, adding new data to the existing [queries](https://v1.faststore.dev/reference/api/queries).
```

Comment:
```suggestion
You will not be able to add other APIs endpoints to your storefront since this may [impact site performance](https://v1.faststore.dev/how-to-guides/faststore-api/fetching-api-data#best-practices-for-fetching-data). If you need to fetch more data, it is possible to extend the FastStore API schema, adding new data to the existing [queries](https://v1.faststore.dev/reference/api/queries).
```